project-base/storefront/components/Basic/ExtendedNextLink/ExtendedNextLink.tsx

```diff
@@ -27,6 +29,7 @@ export const ExtendedNextLink: FC<ExtendedNextLinkProps> = ({
     ...props
 }) => {
     const updatePageLoadingState = useSessionStore((s) => s.updatePageLoadingState);
+    const { url: baseUrl } = useDomainConfig();
```

Comment:
We try not to rename this parameter. Is it necessary?

Reply:
Because I thought just `url` might be confused as the`url` of the current link, but I guess that is `href`, so I will not rename it.